### PR TITLE
Feature/tune 122 adjust login pages

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review when you get a chance :)

screenshot 2016-08-05 10 20 42

**Comment by Mitso** [comment]
screenshot 2016-08-05 10 21 57

File: tuneme/templates/new/registration/login.html [hunk 1]
```diff
@@ -2,28 +2,39 @@
...
-	<a href="{% url 'molo.profiles:user_register' %}">{% trans "Don't have an account? Register" %}</a>
-
-	<form method="post" action="">
-		{% csrf_token %}
+<h1 class="heading heading--profiles">{% trans "Log in" %}</h1>
+<div class="call-to-action call-to-action--profile">
+    <h5 class="call-to-action__nav-item call-to-action__nav-item--challenge">
+    {% trans "Don't have an account yet?" %}<br/>
+    <a href="{% url 'molo.profiles:user_register' %}" class="call-to-action__link">
+    {% trans "Click here" %}</a> {% trans "to join today!" %}</h5>
```
Comment (peterp) [inline tuneme/templates/new/registration/login.html:10]: Mobile phone user's don't click; they tap. Let's reword this sentence to:

`Don't have an account yet? _Join!_`

File: tuneme/templates/new/registration/login.html [hunk 2]
```diff
@@ -2,28 +2,39 @@
...
+			<label for="id_password">{% trans "4 digit pin" %}</label>
+			{{ form.password }}
+			<span class="input-hint">{% trans "(eg: 5656)" %}</span>
+		</fieldset>
 		<input type="submit" value="{% trans "Log in" %}">
 	</form>
 </div>
+
+<div class="call-to-action call-to-action--profile">
+    <h5 class="call-to-action__nav-item call-to-action__nav-item--challenge">
```
Comment (peterp) [inline tuneme/templates/new/registration/login.html:35]: Please make this match the exact same wording above: 

Don't have an account yet? _Join!_

File: tuneme/templates/new/profiles/register.html [hunk 3]
```diff
@@ -3,40 +3,63 @@
...
-
+      {{ form.terms_and_conditions }}
+      <label for="id_terms_and_conditions" class="profiles__terms">{% trans "I accept the Terms and Conditions" %}</label>
+    </fieldset>
     <input type="submit" value="{% trans "Register" %}" />
   </form>
 </div>
+<div class="call-to-action call-to-action--profile">
+    <h5 class="call-to-action__nav-item call-to-action__nav-item--challenge">
+    {% trans "Already registered?" %}</h5>
```
Comment (peterp) [inline tuneme/templates/new/profiles/register.html:61]: Change to "Already a member?"

**Pull request closed without merging**